deploy/ocs-operator/manifests/ocs-operator.clusterserviceversion.yaml

```diff
@@ -3293,12 +3291,17 @@ spec:
                 - -https-address=:8888
                 - -http-address=
                 - -email-domain=*
-                - -upstream=https://localhost:8080/onboarding-tokens
+                - -upstream=https://ux-backend-proxy.openshift-storage.svc:8080/onboarding-tokens
                 - -tls-cert=/etc/tls/private/tls.crt
                 - -tls-key=/etc/tls/private/tls.key
                 - -cookie-secret-file=/etc/proxy/secrets/session_secret
                 - -openshift-service-account=ux-backend-server
                 - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                - -openshift-delegate-urls={"/":{"resource":"namespaces","verb":"get"}}
```

Comment:
this is a weaker restriction, it needs a discussion and I believe a better option is to permit to such SA's that can perform CRUD operation on StorageConsumer